### PR TITLE
first PR

### PR DESCRIPTION
**Pull request by AntonioCarta**
CNN model to filter doublet using hit shapes and info features about the detector layout.
* dataset.py loads the data and prepares it for the neural network
* make_dataset.py split the data in train/val/test set
* doublet_model.py defines the model and the training loop

### CODE REVIEW

**Comment by felicepantaleo** [comment]
@AntonioCarta a readme file is missing

**Comment by AntonioCarta** [comment]
I added a small README file inside the folder
